.github/workflows/scala.yml

```diff
@@ -1,4 +1,9 @@
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - master
+  tags:
+    # run for all tags
```

Comment:
do we want to build this on non-master tags? (or rather: what would change if we removed the `tags:` line?)

Reply:
I don't have a strong opinion on non-master tags; they'd have to be pushed by repo collaborators, so I don't see a reason to prevent them.

Thanks for bringing up the discussion though, it made me realize that the nesting is wrong -- tags is on the same level as `branches`, and specifies a filter of which tag names should trigger a build. I think we're fine to remove this line; after all, the v0.6.0 release worked without it :).